app/i2c.c

```
@@ -0,0 +1,364 @@
+
+    ret = nrf_drv_twi_init(&m_twi_master, &config, twi_handler, NULL);
+
+    if(NRF_SUCCESS == ret)
+    {
+        nrf_drv_twi_enable(&m_twi_master);
+    }
+
+    return ret;
+}
+
+bool i2c_master_write(uint8_t* buf, uint32_t len)
```

Cppcheck (2.10-2)

[style] The function 'i2c_master_write' is never used.

(unusedFunction)